packages/material-ui-lab/src/Skeleton/Skeleton.js

```diff
@@ -18,7 +18,9 @@ export const styles = (theme) => ({
     height: 'auto',
     transformOrigin: '0 60%',
```

Comment:
```suggestion
    transformOrigin: '0 55%',
```

**Before**
<img width="254" alt="Capture d’écran 2020-06-22 à 21 59 49" src="https://user-images.githubusercontent.com/3165635/85333225-8033c200-b4d9-11ea-9bcc-3098013e92c4.png">


**After**
<img width="200" alt="Capture d’écran 2020-06-22 à 22 00 05" src="https://user-images.githubusercontent.com/3165635/85333233-82961c00-b4d9-11ea-8ba6-b8e0521db68d.png">